assets/js/modules/idea-hub/components/dashboard/DashboardCTA/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * DashboardCTA component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import {
+	showErrorNotification,
+} from '../../../../../util';
+import GenericError from '../../../../../components/legacy-notifications/generic-error';
+import { CORE_MODULES } from '../../../../../googlesitekit/modules/datastore/constants';
+import { CORE_LOCATION } from '../../../../../googlesitekit/datastore/location/constants';
+import Button from '../../../../../components/Button';
+import Link from '../../../../../components/Link';
+import IdeaHubIcon from '../../../../../../svg/idea-hub.svg';
+import BulbIcon from '../../../../../../svg/bulb.svg';
+const { useSelect, useDispatch } = Data;
+
+const MODULE_SLUG = 'idea-hub';
```

Comment:
This seems a bit out of place in the component here. Ideally this should go in module-level `constants.js` but even Analytics doesn't have one for the `MODULE_SLUG` yet. Let's inline this for now but it would be great if you could open a new issue for refactoring module slug references with constants.

Reply:
@aaemnnosttv I've created an issue for this - let me know what you think?
https://github.com/google/site-kit-wp/issues/3417